mscore/mixertrackitem.cpp

```diff
@@ -204,6 +204,30 @@ void MixerTrackItem::setColor(int valueRgb)
             }
       }
 
+//---------------------------------------------------------
+//   isMute
+//---------------------------------------------------------
+
+bool MixerTrackItem::isMute(){
+    if (_trackType == TrackType::CHANNEL){
+        return _chan->mute();
+    }else{
+        const InstrumentList* il = _part->instruments();
+        bool muteFound = false;
+
+        for (auto it = il->begin(); it != il->end(); ++it) {
+
+            Instrument* instr = it->second;
+
+            for (Channel *chan: instr->channel()) {
+                muteFound = chan->mute();
```

Comment:
Should this break or return when it finds a mute? What if the list of channels go "true" followed by "false"? Should muteFound be true or false once it finishes the loops?

Reply:
If you only need to check if one is true, just return if true at this point. Then, if it falls through the nested loops without returning, then you can return false. You might be doing that, but it's kinda hard to read the current code formatting :-)

Reply:
As for the exact decision of multiple channels, best ask other developers. I'm just looking at code.